ci/playbooks/edpm_build_images/edpm_image_builder.yml

```diff
@@ -12,7 +12,7 @@
     }}
 
 - name: "Run ci/playbooks/edpm_build_images/edpm_image_builder.yml"
-  hosts: "{{ cifmw_zuul_target_host | default('all') }}"
+  hosts: "{{ cifmw_target_host | default('localhost') }}"
```

Comment:
hmm nope, `cifmw_zuul_host` is correct: it's a thing set for zuul only, because it's a CI playbook. This param was used/created in order to avoid potential collision. Please correct that one.

Reply:
Addressed